<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Connection</class>
 <widget class="QWidget" name="Connection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>942</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="DapGuiWidget" name="Background">
     <property name="cssStyle">
      <string>backgroundcolor</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
<<<<<<< HEAD
       <widget class="DapGuiWidget" name="kelGuiWidget_2" native="true">
        <property name="cssStyle" stdset="0">
=======
       <widget class="DapGuiWidget" name="kelGuiWidget_2">
        <property name="cssStyle">
>>>>>>> features-5100
         <string>conn-top-spacer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="DapGuiLabel" name="lTitle">
        <property name="styleSheet">
         <string notr="true">#lTitle{ 	height: 28px; 	min-height: 28px; 	max-height: 28px; 
 	font: 24px/29px; 
 	color: #DA0B82; 
 	font-weight: bold; 	font-style: bold; 	font-size: bold; 
 	font-family: 'Lato'; 
 	border: none; 
 	background-color: rgba(0,0,0,0); }</string>
        </property>
        <property name="text">
         <string>VPN Connected</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
<<<<<<< HEAD
        <property name="cssStyle" stdset="0">
=======
        <property name="cssStyle">
>>>>>>> features-5100
         <string>conn-top-text-connected font24 red normalbold lato noborder nobackground</string>
        </property>
       </widget>
      </item>
      <item>
<<<<<<< HEAD
       <widget class="DapGuiWidget" name="kelGuiWidget_5" native="true">
        <property name="cssStyle" stdset="0">
=======
       <widget class="DapGuiWidget" name="kelGuiWidget_5">
        <property name="cssStyle">
>>>>>>> features-5100
         <string>conn-top-text-time</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="DapGuiLabel" name="lStatusIconOn">
           <property name="cssStyle">
            <string>conn-status-icon ic_online noborder nobackground</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="DapGuiLabel" name="lStatusIconOff">
           <property name="cssStyle">
            <string>conn-status-icon ic_offline noborder nobackground</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="DapGuiLabel" name="lUptime">
           <property name="styleSheet">
            <string notr="true">#lUptime{ 	height: 22px; 	min-height: 22px; 	max-height: 22px; 
 	font: 18px/22px; 
 	font-weight: normal; 	font-style: normal; 	font-size: normal; 
 	color: #04004E; 
 	font-family: 'Lato'; 
 	border: none; 
 	background-color: rgba(0,0,0,0); }</string>
           </property>
           <property name="text">
            <string>56 : 30 : 10</string>
           </property>
<<<<<<< HEAD
           <property name="cssStyle" stdset="0">
=======
           <property name="cssStyle">
>>>>>>> features-5100
            <string>conn-top-text-time font18 normal darkblue lato noborder nobackground</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
<<<<<<< HEAD
       <widget class="DapGuiWidget" name="kelGuiWidget_3" native="true">
        <property name="cssStyle" stdset="0">
=======
       <widget class="DapGuiWidget" name="kelGuiWidget_3">
        <property name="cssStyle">
>>>>>>> features-5100
         <string>conn-mid-button-spacer</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
<<<<<<< HEAD
         <widget class="DapGuiSwitch" name="btnSwitch" native="true"/>
=======
         <widget class="DapGuiSwitch" name="btnSwitch"/>
>>>>>>> features-5100
        </item>
       </layout>
      </item>
      <item>
<<<<<<< HEAD
       <widget class="DapGuiWidget" name="kelGuiWidget_4" native="true">
        <property name="cssStyle" stdset="0">
=======
       <widget class="DapGuiWidget" name="kelGuiWidget_4">
        <property name="cssStyle">
>>>>>>> features-5100
         <string>conn-down-button-spacer</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
<<<<<<< HEAD
         <widget class="DapGuiButton" name="lDownload" native="true">
          <property name="cssStyle" stdset="0">
           <string>conn-status-button</string>
          </property>
          <property name="mainText" stdset="0">
           <string>120 kb/s</string>
          </property>
          <property name="subText" stdset="0">
           <string>download</string>
          </property>
          <property name="mainCssClass" stdset="0">
           <string>darkblue bold font26 lato</string>
          </property>
          <property name="subCssClass" stdset="0">
=======
         <widget class="DapGuiButton" name="lDownload">
          <property name="cssStyle">
           <string>conn-status-button</string>
          </property>
          <property name="mainText">
           <string>120 kb/s</string>
          </property>
          <property name="subText">
           <string>download</string>
          </property>
          <property name="mainCssClass">
           <string>darkblue bold font26 lato</string>
          </property>
          <property name="subCssClass">
>>>>>>> features-5100
           <string>gray uppercase font12 lato</string>
          </property>
         </widget>
        </item>
        <item>
<<<<<<< HEAD
         <widget class="DapGuiButton" name="lUpload" native="true">
          <property name="cssStyle" stdset="0">
           <string>conn-status-button</string>
          </property>
          <property name="mainText" stdset="0">
           <string>34.7 kb/s</string>
          </property>
          <property name="subText" stdset="0">
           <string>upload</string>
          </property>
          <property name="mainCssClass" stdset="0">
           <string>darkblue normalbold font26 lato</string>
          </property>
          <property name="subCssClass" stdset="0">
=======
         <widget class="DapGuiButton" name="lUpload">
          <property name="cssStyle">
           <string>conn-status-button</string>
          </property>
          <property name="mainText">
           <string>34.7 kb/s</string>
          </property>
          <property name="subText">
           <string>upload</string>
          </property>
          <property name="mainCssClass">
           <string>darkblue normalbold font26 lato</string>
          </property>
          <property name="subCssClass">
>>>>>>> features-5100
           <string>gray uppercase font12 lato</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
<<<<<<< HEAD
         <widget class="DapGuiButton" name="btnServer" native="true">
=======
         <widget class="DapGuiButton" name="btnServer">
>>>>>>> features-5100
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
<<<<<<< HEAD
          <property name="cssStyle" stdset="0">
           <string>conn-connectedto-button nobackground</string>
          </property>
          <property name="mainText" stdset="0">
           <string>Location</string>
          </property>
          <property name="subText" stdset="0">
           <string>000:000:000:000:0000</string>
          </property>
          <property name="mainCssClass" stdset="0">
           <string>darkblue font18 normalbold</string>
          </property>
          <property name="subCssClass" stdset="0">
           <string>gray uppercase font16 lato normal</string>
          </property>
          <property name="leftCssClass" stdset="0">
           <string>darkblue normalbold font30 lato</string>
          </property>
          <property name="link" stdset="0">
           <bool>true</bool>
          </property>
          <property name="frame" stdset="0">
=======
          <property name="cssStyle">
           <string>conn-connectedto-button nobackground</string>
          </property>
          <property name="buttonStyle" stdset="0">
           <enum>DapGuiButton::TopSubBottomMain</enum>
          </property>
          <property name="mainText">
           <string>North America - Canada</string>
          </property>
          <property name="subText">
           <string>231:532:531:20</string>
          </property>
          <property name="mainCssClass">
           <string>darkblue font18 normalbold</string>
          </property>
          <property name="subCssClass">
           <string>gray uppercase font16 lato normal</string>
          </property>
          <property name="leftCssClass">
           <string>darkblue normalbold font30 lato</string>
          </property>
          <property name="link">
           <bool>true</bool>
          </property>
          <property name="frame">
>>>>>>> features-5100
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
<<<<<<< HEAD
       <widget class="DapGuiWidget" name="kelGuiWidget" native="true">
        <property name="cssStyle" stdset="0">
=======
       <widget class="DapGuiWidget" name="kelGuiWidget">
        <property name="cssStyle">
>>>>>>> features-5100
         <string>footer</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DapGuiButton</class>
   <extends>QWidget</extends>
   <header>dapguibutton.h</header>
  </customwidget>
  <customwidget>
   <class>DapGuiSwitch</class>
   <extends>QWidget</extends>
   <header>dapguiswitch.h</header>
  </customwidget>
  <customwidget>
   <class>DapGuiLabel</class>
   <extends>QLabel</extends>
   <header>dapguilabel.h</header>
  </customwidget>
  <customwidget>
   <class>DapGuiWidget</class>
   <extends>QWidget</extends>
   <header>dapguiwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
